charts/service-deployment/Chart.yaml

```diff
@@ -1,7 +1,7 @@
 apiVersion: v2
 name: service-deployment
 description: A Helm Chart to setup a generic deployment with optional service/hpa bindings
-version: 0.4.1
+version: 0.4.2
```

Comment:
@jparavisini if this is changing to a new autoscaling major version (v2) feels like this should be a 0.5.0 not a revision bump?

Reply:
Agreed, good call!